tests/docker_compose/custom-gateway/dummy_gateway.py

```diff
@@ -21,16 +21,11 @@ class ProcessedResponseModel(BaseModel):
 
 class DummyGateway(Gateway):
     def __init__(
-        self,
-        port: int = None,
-        host: str = __default_host__,
-        arg1: str = None,
-        arg2: str = None,
-        arg3: str = 'default-arg3',
-        **kwargs
+        self, arg1: str = None, arg2: str = None, arg3: str = 'default-arg3', **kwargs
     ):
         super().__init__(**kwargs)
         self.port = self.runtime_args.port[0]
+        self.host = self.runtime_args.host
```

Comment:
for the `gateway` we should provide these properties:

```python
@property
def port():
    return self.runtime_args.port[0]

@property
def ports():
    return self.runtime_args.port

@property
def host():
    return self.runtime_args.host
```

Reply:
This we can do in a separate PR

Reply:
okay makes sense to me

Reply:
> Host should be a `runtime_arg`. Also test what happens when a stupid host is passed

added